IS7024Project/Pages/Index.cshtml.cs

```diff
@@ -28,17 +29,16 @@ public void OnGet()
             {
                 //Consuming Parks data
                 string parksJSON = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/Parks_Data.txt");
-                var park = Parks.FromJson(parksJSON);
-                ViewData["Parks"] = park;
+                var parks = Parks.FromJson(parksJSON);
+                ViewData["Parks"] = parks;
                 QuickTypeParks.Parks[] Parksparks = QuickTypeParks.Parks.FromJson(parksJSON);
 
 
                 //Consuming Crime data
-                string jsonString = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/PDI_Crime_Data.txt");
-                var crime = Crime.FromJson(jsonString);
-                ViewData["Crime"] = crime;
+                string crimesJSON = webClient.DownloadString("https://raw.githubusercontent.com/JMFrank215/IS7024Project/master/PDI_Crime_Data.txt");
```

Comment:
changed to crimesJSON for more consistency and to better understand exactly which API and JSON schema you're working with.